src/js/actions/search/adobestock.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports) {
+    "use strict";
+
+    var Immutable = require("immutable");
+
+    var events = require("js/events"),
+        locks = require("js/locks");
+
+    var searchActions = require("js/actions/search");
+    
+    /**
+     * Captures search term for Adobe Stock and executes search
+     *
+     * @private
+     * @param {string} searchTerm
+    */
+    var _confirmStockSearch = function (searchTerm) {
+        searchActions.executeStockSearch.call(this, searchTerm);
+    };
+
+    /**
+     * Find SVG class for Adobe Stock
+     * If this needs to vary based on the item, use category list as parameter 
+     * (see getSVGCallback type in search store)
+     * 
+     * @return {string}
+    */
+    var _getSVGClass = function () {
+        return "libraries-stock";
+    };
+
+    /**
+     * Make list for Stock options, currently empty
+     * 
+     * @private
+     * @return {Immutable.List.<object>}
+    */
+    var _adobeStockSearchOptions = function () {
```

Comment:
The reasons I kept this as a function were: 

1.) In the case that we expand on the functionality of Adobe Stock search in the future, we will have options that will populate through this function. I saw this as an intermediate state which returns an empty list now, that is not unique now, but later on we will end up changing and wanting this function back.

2.) For every registered search payload, payload.getOptions is a function and previously when I had the `getOptions` property be an empty list instead of a function call, it would run into errors at different points in the program when `getOptions()` was called instead of `getOptions`. We can go through and have it check for function or value, etc. but a.) only the stock module requires this and b.) like I said above, I anticipate changing the return value of the function so we would change any calls to `getOptions` in the future back to the original `getOptions()`